### ISSUE TITLE
Exception 'NoneType' object not callable, Nodes.py line 124

### ISSUE THREAD

**Issue by scherma** [williballenthin/python-evtx#32]
Got an issue with the Sysmon event log I'm trying to parse:

```
Traceback (most recent call last):
  File "/usr/local/bin/evtx_dump", line 4, in <module>
    __import__('pkg_resources').run_script('python-evtx==0.5.3', 'evtx_dump')
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 738, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 1506, in run_script
    exec(script_code, namespace, namespace)
  File "/usr/local/lib/python2.7/dist-packages/python_evtx-0.5.3-py2.7.egg/EGG-INFO/scripts/evtx_dump", line 53, in <module>

  File "/usr/local/lib/python2.7/dist-packages/python_evtx-0.5.3-py2.7.egg/EGG-INFO/scripts/evtx_dump", line 48, in main

  File "build/bdist.linux-x86_64/egg/Evtx/Views.py", line 218, in evtx_file_xml_view
  File "build/bdist.linux-x86_64/egg/Evtx/Views.py", line 169, in _build_record_xml
  File "build/bdist.linux-x86_64/egg/Evtx/Views.py", line 160, in rec
  File "build/bdist.linux-x86_64/egg/Evtx/Views.py", line 154, in rec
  File "build/bdist.linux-x86_64/egg/Evtx/Views.py", line 129, in _make_template_xml_view
  File "build/bdist.linux-x86_64/egg/Evtx/BinaryParser.py", line 64, in __call__
  File "build/bdist.linux-x86_64/egg/Evtx/Nodes.py", line 139, in children
  File "build/bdist.linux-x86_64/egg/Evtx/Nodes.py", line 124, in _children
TypeError: 'NoneType' object is not callable
```

I can try playing around with the source when I get home tonight to see if I can persuade it to give me the exact bytes it's failing on; I can also provide you with the event data - it works fine in EventViewer - but would prefer to send that privately rather than posting it here. Let me know!

**Comment by williballenthin** [comment]
if you can share the source data, i'd be happy to triage the issue. you can email me at wilbal1087@gmail.com

**Comment by scherma** [comment]
Unfortunately it's in a VM and I forgot to take a copy before restoring... my current evtx is fine, I'm going to have to do a few more runs and try to reproduce it. Will let you know if I can't, it may just have been a bad write within the VM.

**Comment by williballenthin** [comment]
i found this issue today (ultimately character entity references weren't supported), and have fixed it. you should no longer encounter the issue from v0.6.0 onwards!

closed in 5ec3136c60323bfc0d276393d44babdfc58bfa6b
